cmd/zetae2e/config/config.go

```diff
@@ -67,7 +67,8 @@ func ExportContractsFromRunner(r *runner.E2ERunner, conf config.Config) config.C
 
 	conf.Contracts.Sui.FungibleTokenCoinType = config.DoubleQuotedString(r.SuiTokenCoinType)
 	conf.Contracts.Sui.FungibleTokenTreasuryCap = config.DoubleQuotedString(r.SuiTokenTreasuryCap)
-	conf.Contracts.Sui.Example = r.SuiExample
+	conf.Contracts.Sui.ExampleArbiCall = r.SuiExampleArbiCall
```

Comment:
There might have been some misunderstanding there but we will not need to maintain "arbitrary call" test for Sui, the message context will always be passed to the call therefore, all calls will be "authenticated"

Reply:
To confirm, we have the EVM chain gateway [execute](https://github.com/zeta-chain/protocol-contracts/blob/main/contracts/evm/GatewayEVM.sol#L153) method making two types of calls on client contract. In `sui`, we only need the second one? like `_executeAuthenticatedCall`?

```
if (messageContext.sender == address(0)) {
     result = _executeArbitraryCall(destination, data);
} else {
    result = _executeAuthenticatedCall(messageContext, destination, data); // only need this for Sui?
}
```